src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java

```diff
@@ -201,15 +201,12 @@ public void setChatUser(ChatChannelUser chatUser) {
       removeFoeItem.visibleProperty().bind(newValue.socialStatusProperty().isEqualTo(FOE));
       reportItem.visibleProperty().bind(newValue.socialStatusProperty().isNotEqualTo(SELF));
 
-      // TODO: Make this ignore TMM games too and not just ladder
-      // https://github.com/FAForever/downlords-faf-client/issues/1770
       joinGameItem.visibleProperty().bind(newValue.socialStatusProperty().isNotEqualTo(SELF)
           .and(newValue.statusProperty().isEqualTo(PlayerStatus.LOBBYING)
               .or(newValue.statusProperty().isEqualTo(PlayerStatus.HOSTING)))
           .and(Bindings.createBooleanBinding(() -> {
                 return newValue.getGame() != null
-                    && newValue.getGame().getFeaturedMod() != null
-                    && !newValue.getGame().getFeaturedMod().equals(KnownFeaturedMod.LADDER_1V1.getTechnicalName());
+                    && newValue.getGame().getGameType() != GameType.MATCHMAKER;
```

Comment:
Just ladder not be excluded as well

Reply:
Ladder falls into the category of matchmaker games